slack-api-model/src/test/java/test_locally/api/model/dialog/DialogsTest.java

```diff
@@ -26,4 +27,21 @@ public void test() {
         });
         assertThat(dialog, is(notNullValue()));
     }
+
+    @Test
+    public void testDataSource() {
+        Dialog dialog = dialog(d -> {
+            return d.elements(asElements(
+                    dialogSelect(ds ->
+                            ds.label("aaa")
+                                    .name("nnnn")
+                                    .dataSource(DialogDataSourceType.CONVERSATIONS)
+                    ),
+                    dialogText(dt -> dt.name("foo")),
+                    dialogTextArea(dta -> dta.subtype(DialogSubType.NUMBER))
+            ));
+        });
+        System.out.println(dialog.toString());
```

Comment:
Remove this stdout printing. It's ideal if we could add more tests for external data source. But it's totally fine to merge your contribution without it. I can work on it in addition to yours afterward.